here-naksha-lib-core/src/main/java/com/here/naksha/lib/core/models/geojson/implementation/namespaces/XyzNamespace.java

```diff
@@ -23,22 +23,25 @@
 import com.fasterxml.jackson.annotation.JsonInclude.Include;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.here.naksha.lib.core.util.json.JsonObject;
-import java.text.Normalizer;
-import java.text.Normalizer.Form;
-import java.util.ArrayList;
-import java.util.List;
 import naksha.geo.XyzProperties;
 import naksha.model.EXyzAction;
 import naksha.model.NakshaVersion;
 import org.jetbrains.annotations.ApiStatus.AvailableSince;
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
 
+import java.text.Normalizer;
+import java.text.Normalizer.Form;
+import java.util.ArrayList;
+import java.util.List;
+
 /**
  * The properties stored as value for the {@link XyzProperties#XYZ_NAMESPACE @ns:com:here:xyz} key in the {@link XyzProperties properties}
  * of features managed by Naksha. Except for the {@code tags} all these values are read-only.
+ * @deprecated please use {@link naksha.model.XyzNs} instead.
  */
 @SuppressWarnings("unused")
+@Deprecated
```

Comment:
If it's possible, maybe we should simply delete this?

Reply:
As discussed in group, we keep this as backup, to make sure we have migrated everything to the new class.